src/goto-instrument/contracts/contracts.cpp

```diff
@@ -852,6 +854,27 @@ void code_contractst::apply_function_contract(
     new_program.destructive_append(assertion);
   }
 
+  // Translate requires_contract(ptr, contract) clauses to assertions
+  // ASSERT ptr == contract;
+  for(auto &expr : requires_contract)
+  {
+    auto &obeys_expr = to_function_pointer_obeys_contract_expr(expr);
+    replace_symbolt replace(common_replace);
```

Comment:
I don't think you need to take a copy here, just use `common_replace` directly two lines later.

Reply:
fixed